### PR TITLE
Fix composer memory limit issue

### PR DESCRIPTION
**Pull request by fitn**

### CODE REVIEW

**Comment by phaseinducer** [comment]
GTM
Thanks dude :+1:

**Comment by jmleroux** [comment]
GTM

**Comment by nidup** [comment]
no need for other use of composer in the doc? install etc?

File: developer_guide/installation/setup_hybrid_storage_mysql_mongo.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ Installing and enabling MongoDB support in Akeneo
     :linenos:
 
     $ cd /path/to/installation/pim-community-standard
-    $ php ../composer.phar --prefer-dist require doctrine/mongodb-odm-bundle 3.0.1
+    $ php -d memory_limit=3G ../composer.phar --prefer-dist require doctrine/mongodb-odm-bundle 3.0.1
```
Comment (phaseinducer) [inline developer_guide/installation/setup_hybrid_storage_mysql_mongo.rst]: Not related to your change but it seems that the `../composer.phar` has an extra dot.

**Pull request merged**